src/token-group/token.tsx

```diff
@@ -6,25 +6,48 @@ import clsx from 'clsx';
 
 import DismissButton from './dismiss-button';
 import styles from './styles.css.js';
+import { FormFieldError } from '../form-field/internal';
+import { useUniqueId } from '../internal/hooks/use-unique-id';
 
 interface TokenProps {
   children: React.ReactNode;
   ariaLabel?: string;
   dismissLabel?: string;
   onDismiss?: () => void;
   disabled?: boolean;
+  errorText?: React.ReactNode;
+  errorIconAriaLabel?: string;
+  className?: string;
 }
 
-export function Token({ ariaLabel, disabled, dismissLabel, onDismiss, children }: TokenProps) {
+export function Token({
+  ariaLabel,
+  disabled,
+  dismissLabel,
+  onDismiss,
+  children,
+  errorText,
+  errorIconAriaLabel,
+  className,
+}: TokenProps) {
+  const errorId = useUniqueId('error');
   return (
     <div
-      className={clsx(styles.token, disabled && styles['token-disabled'])}
+      className={clsx(styles.token, className)}
       role="group"
       aria-label={ariaLabel}
+      aria-describedby={errorText ? errorId : undefined}
       aria-disabled={disabled}
     >
-      {children}
-      {onDismiss && <DismissButton disabled={disabled} dismissLabel={dismissLabel} onDismiss={onDismiss} />}
+      <div className={clsx(styles['token-box'], disabled && styles['token-box-disabled'])}>
+        {children}
+        {onDismiss && <DismissButton disabled={disabled} dismissLabel={dismissLabel} onDismiss={onDismiss} />}
+      </div>
+      {errorText && (
+        <FormFieldError id={errorId} errorIconAriaLabel={errorIconAriaLabel}>
+          {errorText}
+        </FormFieldError>
+      )}
```

Comment:
How about making it a slot, to the consumer of the token will render `FormFieldError` when needed.

Reply:
Generally I don't mind it but it comes with a downside. The error association would have to be done on the outside:

```
function Demo({ tokens }) {
  return <>{tokens.map((token, index) => (
    <Token ... ariaDescription={`${errorId}-${index}`} error={<FormFieldError id={`${errorId}-${index}`}>{token.error}</FormFieldError>}>{token.content}</Token>
  ))}</>
}
```

Reply:
See the comment above https://github.com/cloudscape-design/components/pull/829/files?w=1#r1164379793. You can avoid any complexity in token group at all.

Not everything in this world needs to be `describedby`

Reply:
Apart from that, what is the benefit of having it as a slot? Do we expect different error designs for the token?